doc/examples/applications/plot_tool_using_holoviz.py

```diff
@@ -0,0 +1,189 @@
+"""
+=======================================
+Interactive Tools and Apps with HoloViz
+=======================================
+
+This tutorial shows how to build interactive tools and applications
+using `Panel <https://panel.holoviz.org>`_, `HoloViews <https://holoviews.org>`_ you can use to make you and your teams
+analysis more interactive and powerful.
+
+.. figure:: https://user-images.githubusercontent.com/42288570/146666639-02f0106f-a3e2-4306-9033-707554e4e61e.gif
+   :alt: A video showcasing the application
+
+   A video showcasing the application
+
+You can develop using notebooks or .py files. You can use it with a very
+broad range of Pythons Viz. You can use the tools in a notebook, your editor/ IDE or
+deploy to a web server.
+
+To run this example you would need to get some python packages
+installed. You can create an environment from scratch using.
+
+.. code:: bash
+
+   conda create -n scikit-image-demo -c conda-forge jupyterlab panel holoviews scikit-image selenium \
+   firefox geckodriver
+   conda activate scikit-image-demo
+   jupyter serverextension enable panel.io.jupyter_server_extension
+"""
+
+
+############################
+# Import packages
+# ==========================
+
+
+from io import BytesIO
+
+import holoviews as hv
+import panel as pn
+from skimage import data, filters
+
+#############################################
+# Activate the Panel and HoloViews extensions

```

Comment:
```suggestion
# Activate the Panel and HoloViews extensions
# ==================================
```
(looks like this should be a section title, hierarchically at the same level as "Import packages")

Reply:
I think that all three above 'sections' (set up env, import packages, and activate extensions) could/should go under one single section.